CoreDependencies/src/ivy.xml

```diff
@@ -81,6 +81,8 @@
         <dependency org="org.python" name="jython-standalone" rev="2.7.2"/>
         <dependency org="org.swinglabs" name="swingx" rev="1.6.1"/>
         <dependency org="org.fxmisc.richtext" name="richtextfx" rev="0.10.6"/>
+        <dependency org="org.languagetool" name="language-en" rev="5.7"/>
+        <dependency org="tech.units" name="indriya" rev="1.3" force="true" />
```

Comment:
I note from the project.xml that we already had v2.0.2 of this. Was there a reason why this specific version was needed?

Reply:
Yes, remember it took long to get the LanguageTool libworking in Consty, it was causing issues with v2 of indriya . Arcturus2 identified the problem. pom.xml mentions 1.3 of indriya but our ivy was getting v2. Also the force="true" was also needed.